src/kava-model.js

```diff
@@ -30,6 +30,7 @@ class KavaModel {
   droppedFramesRatio: ?number = null;
   soundMode: typeof SoundMode;
   tabMode: typeof TabMode;
+  screenMode: number = ScreenMode.NOT_IN_FULLSCREEN;
```

Comment:
you can use https://flow.org/en/docs/types/utilities/#toc-keys and https://flow.org/en/docs/types/utilities/#toc-values